connectors/sources/notion.py

```diff
@@ -423,7 +465,7 @@ def get_default_configuration(cls):
                 "value": False,
             },
             "concurrent_downloads": {
-                "default_value": 20,
+                "default_value": 30,
```

Comment:
Why's this being changed?

Reply:
same as the reason above, it was changed for file downloads.